stable/elasticsearch-exporter/templates/service.yaml

```diff
@@ -19,6 +19,10 @@ spec:
   ports:
     - name: {{ .Values.service.metricsPort.name }}
       port: {{ .Values.service.httpPort }}
+      {{- if eq .Values.service.type "NodePort"}}
+      targetPort: {{ .Values.service.targetPort }}
```

Comment:
I'm curious: what is the use case where the port is different than the targetPort?

Note that kube doc reads `If this is not specified, the value of the 'port' field is used (an identity map)`